File: app/src/main/java/com/mediaplayer/listeners/VolumeOnSeekBarChangeListener.java
package com.mediaplayer.listeners;

import android.widget.SeekBar;

import com.mediaplayer.services.MediaControl;
import com.mediaplayer.variables.CommonArgs;

public class VolumeOnSeekBarChangeListener implements SeekBar.OnSeekBarChangeListener {

    Boolean isViewOn = false;
    int Progress;

    @Override
    public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
        Progress = progress;
        new MediaControl().setVolume(Progress);
        isViewOn = false;
    }

    @Override
    public void onStartTrackingTouch(SeekBar seekBar) {

    }

    @Override
    public void onStopTrackingTouch(SeekBar seekBar) {
        new MediaControl().setVolume(Progress);
        isViewOn = true;
//        CommonArgs.autoFade(CommonArgs.rl_volume_seekbar, isViewOn);
    }

}

File: app/src/main/java/com/mediaplayer/listeners/VolumeOnSeekBarChangeListener.kt
package com.mediaplayer.listeners

import android.widget.SeekBar

import com.mediaplayer.services.MediaControl
import com.mediaplayer.variables.CommonArgs

class VolumeOnSeekBarChangeListener : SeekBar.OnSeekBarChangeListener {

    private var isViewOn: Boolean? = false
    private var progress: Int = 0

    override fun onProgressChanged(seekBar: SeekBar, progress: Int, fromUser: Boolean) {
        this.progress = progress
        MediaControl().setVolume(progress)
        isViewOn = false
    }

    override fun onStartTrackingTouch(seekBar: SeekBar) {

    }

    override fun onStopTrackingTouch(seekBar: SeekBar) {
        MediaControl().setVolume(this.progress)
        isViewOn = true
        //        CommonArgs.autoFade(CommonArgs.rl_volume_seekbar, isViewOn);
    }

}
